Welcome dialogue for 5.8

dereksmart opened the pull request:
This will add a welcome dialogue to the 5.8 release. 

To Test: 
- Manually change the `JETPACK__VERSION` constant in jetpack.php to 5.8
- Load the Jetpack admin dashboard
- See the dialogue
- Should not be there on reload.  

~(in progress until we have the copy)~
(in progress until we have updated images)

jeffgolenski commented:
Alright. I made a few spacing changes and will work on getting the imagery needed!

dereksmart commented:
Removed mention of AL/R and added a temp screenshot that will be updated during beta

Looks like this now
![screen shot 2018-01-30 at 1 31 47 pm](https://user-images.githubusercontent.com/7129409/35584116-035dcd7a-05c2-11e8-81d9-f594d1b3d7f3.png)
